tools/sample_generator/templates/default/WinUI.xaml

```diff
@@ -0,0 +1,10 @@
+<UserControl 
+    x:Class="ArcGISRuntime.WinUI.Samples.sample_name.sample_name"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:esri="using:Esri.ArcGISRuntime"
+    xmlns:esriUI="using:Esri.ArcGISRuntime.UI.Controls">
+    <Grid>
+        <esriUI:Geo_View x:Name="MyGeo_View" />
```

Comment:
Is `Geo_View` a placeholder?

Reply:
Geo_View gets changed to MapView or SceneView when running the script.